shared/settings/landing/container.js

```diff
@@ -1,53 +1,64 @@
 // @flow
-import * as actions from '../../actions/plan-billing'
+import * as actions from '../../actions/settings'
 import Bootstrapable from '../../util/bootstrapable'
 import Landing from './index'
 import {connect} from 'react-redux'
 import {routeAppend} from '../../actions/router'
 
 import type {TypedState} from '../../constants/reducer'
 
-type OwnProps = {}
-
 export default connect(
-  (state: TypedState, ownProps: OwnProps) => {
+  (state: TypedState, ownProps: {}) => {
+    const {emails} = state.settings.email
+    let accountProps
+    if (emails.length > 0) {
+      accountProps = {
+        email: emails[0].email,
+        isVerified: emails[0].isVerified,
+        onChangeEmail: () => console.log('todo'),
+        onChangePassphrase: () => console.log('todo'),
+      }
+    }
+
     const {planBilling: {availablePlans, usage, plan, paymentInfo}} = state
-    if (!availablePlans || !usage || !plan) {
-      return {
-        bootstrapDone: false,
+    let planProps
+    if (plan && usage) {
+      const freeSpaceGB = plan.gigabytes - usage.gigabytes
+      const freeSpacePercentage = freeSpaceGB / plan.gigabytes
+      planProps = {
+        onUpgrade: () => console.log('todo'),
+        onDowngrade: () => console.log('todo'),
+        onInfo: () => console.log('todo'),
+        selectedLevel: plan.planLevel,
+        freeSpace: freeSpaceGB + 'GB',
+        freeSpacePercentage,
+        lowSpaceWarning: false,
+        paymentInfo,
+        onChangePaymentInfo: () => console.log('todo'),
       }
     }
 
-    const freeSpaceGB = plan.gigabytes - usage.gigabytes
-    const freeSpacePercentage = freeSpaceGB / plan.gigabytes
+    // When enabling planProps, we should check both for bootstrapDone:
+    // let bootstrapDone = accountProps && planProps
+    let bootstrapDone = !!accountProps
```

Comment:
Currently it only waits for account props cause plans aren't being loaded anymore (that was temporarily removed from view).